mitxgraders/listgrader.py

```diff
@@ -734,3 +741,33 @@ def check_response(self, answer, student_input, **kwargs):
                                         partial_credit=self.config['partial_credit'])
 
         return result
+
+def demand_no_nonempty(obj):
```

Comment:
should this be `demand_all_nonempty`? or `demand_no_empty`?

Reply:
Oops! Fixed.